### PR TITLE
Commented out variables still end up in the output

### LINKED ISSUE #7: Commented out variables still end up in the output
This was a major source of confusion for me. 

If I comment out something in my less file, the variables still end up in the output. So less-vars-to-js seems to basically ignore them all together.

This goes for /* */ blocks and // inline comments.

### PR DESCRIPTION
**Pull request by jean343**
Fixes #7 

Also,
- Will support SASS

Also with options,
- The variables will be resolved from other values in the same file, or from a supplied object.
- The resulting object will not contain the `@` or `$` character.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "less-vars-to-js",
-  "version": "1.2.1",
+  "version": "1.3.0",
```
Comment (michaeltaranto) [inline package.json]: Can you roll this back, i'll handle the versioning in a separate commit.

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1,41 @@
-export default sheet => {
-  const lessVars = {};
-  const matches = sheet.match(/@(.*:[^;]*)/g) || [];
+import stripComments from 'strip-json-comments';
+
+const varRgx = /^[@$]/;
+const followVar = (value, lessVars, defaults) => {
```
Comment (michaeltaranto) [inline src/index.js]: `defaults` doesnt feel like the right name for this. But it's a tricky one. Im thinking something like `dictionary`? Did you have any other ideas?

**Comment by jean343** [comment]
Fair comments, I took `dictionary` as my only other option was `fallback`.

**Review by michaeltaranto — APPROVED** [review]
Thanks for making those changes 👏

**Pull request merged**